lib/helpers.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var join = require('path').join;
+
+module.exports = {
+  expandHomeDir: function (path) {
```

Comment:
Please name functions of this importance, so it's easier to debug in the future. And if you would rename this file to only contain one helper function, you could export that function only.